src/shim/actors/builtin/market/mod.rs

```diff
@@ -584,7 +602,10 @@ impl TryFrom<&fil_actor_market_state::v15::DealProposal> for DealProposal {
             provider: from_address_v4_to_v2(deal_proposal.provider),
             label: match &deal_proposal.label {
                 fil_actor_market_state::v15::Label::String(s) => s.clone(),
-                fil_actor_market_state::v15::Label::Bytes(b) => String::from_utf8(b.clone())?,
+                fil_actor_market_state::v15::Label::Bytes(b) if b.is_empty() => Default::default(),
+                fil_actor_market_state::v15::Label::Bytes(b) => {
+                    String::from_utf8(b.clone()).unwrap_or_default()
```

Comment:
Tried `String::from_utf8_lossy` and `String::from_utf8_unchecked` but neither worked.